var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -31,6 +34,12 @@ class Esmf(MakefilePackage):
     variant("netcdf", default=True, description="Build with NetCDF support")
     variant("pnetcdf", default=True, description="Build with pNetCDF support")
     variant("xerces", default=True, description="Build with Xerces support")
```

Comment:
We could make xerces by default False since ESMf is moving to Info object.

Reply:
done